packages/verification-app/src/App.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div id="app">
+    <div id="app" class="app">
```

Comment:
You could call the class "cerification-app" to make it consistent with the package name.

Reply:
Changed it to verification-app